### PR TITLE
More explanations on adb install

### PR DESCRIPTION
**Pull request by a14n**

### CODE REVIEW

**Comment by a14n** [comment]
Related to #87

File: getting-started.md [hunk 1]
```diff
@@ -100,6 +106,8 @@ Jelly Bean, v16, 4.1.x or later.
  - Using a USB cable, plug your phone into your computer. If prompted on your
    device, authorize your computer to access your device.
 
+ - Check that _abd_ lists your device with `adb devices -l`.
```
Comment (abarth) [inline getting-started.md]: _abd_ <--- typo ?

Comment (a14n) [inline getting-started.md]: fixed. thanks :)

**Comment by abarth** [comment]
It looks like we were using `adb` rather than _adb_ to name the tool previously.  We should be consistent about how we name the tool.  Maybe we should use `adb` in the new text as well?  We also use `this style` to refer to text in the Android UI whereas you're using _this style_ for some of the new text.  We should also be consistent with this sort of information...  I think _this style_ is better for the UI strings, so maybe we should change "Android debugging" and similar uses too.

The important thing is to be consistent throughout the document.  (Also, please squash your commits so I can merge them cleanly into master.)

Thanks!!

**Comment by a14n** [comment]
Actually simple backticks mean inline code format. So I'd prefer _Android SDK Manager_ rather than `Android SDK Manager`.

But I agree that consistancy is more important so I've squashed my PR with your recommandations.

File: getting-started.md [hunk 2]
```diff
@@ -91,6 +91,12 @@ Jelly Bean, v16, 4.1.x or later.
    - Linux: `sudo apt-get install android-tools-adb`
      - If the version of `adb` provided by your Linux distribution is too old,
        you might need to [install the Android SDK manually](https://developer.android.com/sdk/installing/index.html?pkg=tools).
+       Note that `adb` needs 32-bit libs (on ubuntu 64-bit, make sure to install
```
Comment (abarth) [inline getting-started.md]: s/ubuntu/Ubuntu/

File: getting-started.md [hunk 3]
```diff
@@ -91,6 +91,12 @@ Jelly Bean, v16, 4.1.x or later.
...
+       those libs listed in _Troubleshooting Ubuntu_ section).
```
Comment (abarth) [inline getting-started.md]: s/libs/libraries/

**Comment by abarth** [comment]
LGTM

**Comment by a14n** [comment]
Let me wake up :)
Thanks!

**Comment by abarth** [comment]
No worries.  I just made the trivial edits after merging your patch.  :)

**Pull request merged**